save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -95,6 +96,23 @@ class ProjectController(
             projectPermissionEvaluator.hasPermission(authentication, it, Permission.READ)
         }
 
+    @PostMapping("/not-deleted-filter")
```

Comment:
It can still be called `/not-deleted`, these methods will differ by request method. Then they even can be merged if we have default value for filters null or empty

Reply:
Ok. Merged these methods into one.